src/components/ContributionsEpic.tsx

```diff
@@ -213,7 +215,7 @@ export const ContributionsEpic: React.FC<Props> = ({
                         Support The Guardian
                     </Button>
                 </div>
-                {secondaryCta && (
+                {secondaryCta && secondaryCta.text && (
```

Comment:
Is there maybe an argument that this should happen in a separate PR? If it's possible for the object to exist but not have `text`, is the same true of `baseUrl`? (i.e. do we need to check that too?)

Reply:
Yeah I should have done this on a separate PR. But I don't think it's worth redoing that now. I checked the Frontend logic and they do as you see here - only render the button if the 3 properties exist (the wrapping object plus both text and link.